test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -538,7 +538,7 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             var content = await response.Content.ReadAsStringAsync();
             Assert.StartsWith("Message: Secret post", content.Trim());
-            Assert.EndsWith("TempData:", content.Trim());
+            Assert.EndsWith("TempData: Secret post", content.Trim());
```

Comment:
This was reading a value set by `[TempData]` property in a view. This now works.